cluster/common/kubediff/deploy_kubediff.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
maybe add a rm-rf /kubediff to remove kubediff content before shell scripts run

Reply:
Done